concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -1867,7 +1867,7 @@ jobs:
                 - -e
                 - -c
                 - |
-                  if [ -n "${TF_VAR_datadog_api_key}" -a -n "${TF_VAR_datadog_app_key}" -a "${ENABLE_DATADOG}" = "false" ]; then
+                  if [ -n "${TF_VAR_datadog_api_key}" ] && [ -n "${TF_VAR_datadog_app_key}" ] && [ "${ENABLE_DATADOG}" = "false" ]; then
```

Comment:
what exactly is the shellcheck error? Is this correctly evaluated for 'sh' ? the difference here is that && and || are evaluated by shell, vs -a and -o being evaluated by test...